components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIProvider.java

```diff
@@ -53,23 +52,50 @@ public void deleteBusinessOwner(String businessOwnerId) throws
 
      /* *
      *Get business ower data.
-     * @param ownerId
+     * @param businessOwnerId
      * @return
      * @throws AppManagementException
      */
-    public Map<String, String> getBusinessOwnerCustomProperties(int ownerId) throws AppManagementException;
+    public List<BusinessOwnerProperty> getBusinessOwnerCustomProperties(int businessOwnerId) throws
+                                                                                             AppManagementException;
     /**
      * @return
      * @throws AppManagementException
      */
     public List<BusinessOwner> getBusinessOwners() throws AppManagementException;
 
     /**
+     * Return the owner properties of the given owner Id.
+     *
+     * @param businessOwnerId Business owner Id.
+     * @return
+     * @throws AppManagementException if failed to get business owner.
+     */
+    public BusinessOwner getBusinessOwner(int businessOwnerId) throws AppManagementException;
+
+    /**
+     * Search the business owners with page limitation.
+     * @param startIndex
+     * @param pageSize
+     * @param searchValue
+     * @return
+     * @throws AppManagementException
+     */
+    public List<BusinessOwner> searchBusinessOwners(int startIndex, int pageSize, String searchValue) throws
+                                                                                              AppManagementException;
+
+    /**
+     * Get the count of business owners.
+     * @return
+     * @throws AppManagementException
+     */
+    public  int getBusinessOwnersCount() throws AppManagementException;
+    /**
      * Save a Business Owner.
      * @return Integer
      * @throws AppManagementException
      */
-    public void saveBusinessOwner(BusinessOwner businessOwner) throws AppManagementException;
+    public int saveBusinessOwner(BusinessOwner businessOwner) throws AppManagementException;
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/665ba5bee36f67c4e163ac0422175545ad3a03c5